src/builtins/filters/string.rs

```diff
@@ -411,7 +412,7 @@ mod tests {
         assert!(result.is_err());
         assert_eq!(
             result.err().unwrap().to_string(),
-            "Filter `upper` was called on an incorrect value: got `50` but expected a String"
+            "Filter `upper` was called on an incorrect value: got `50` but expected a alloc::string::String"
```

Comment:
sounds like a regression in terms of error messages

Reply:
we could use [tynm](https://github.com/azriel91/tynm)
tynm shortens the Type to e.g. `Option<String>`

would it be acceptable to add tynm as dependency?